Installation/include/CGAL/license/Polygon_mesh_processing/interpolated_corrected_curvature_measures.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016  GeometryFactory SARL (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s) : Andreas Fabri
```

Comment:
not me.

Reply:
@sloriot what should it be here?

Reply:
You are right. I didn't realize that it is only the license checking file.